### DISCUSSION TITLE
OBB Support

### DISCUSSION THREAD

**Discussion by divyansh-c** [obss/sahi#910] [Q&A] [answered]
I have a YOLOv5 model trained for Oriented Bounding Box detection, will SAHI support the OBB detection?

**Comment by SirojbekSafarov** [comment]
I also wonder if will SAHI support Obb detection model trained on mmrotate.

**Reply by fcakyon** [reply]
@SirojbekSafarov its supported for ultralytics yolov5u/yolov8/yolo11 obb models with latest sahi version: https://github.com/obss/sahi/releases/tag/0.11.20

Feel free to open a PR for mmrotate support. You can take Ultralytics OBB implementation as reference: https://github.com/obss/sahi/blob/4784b77e40a17ab8e83939c8e088257efcf72d52/sahi/models/ultralytics.py#L87

**Comment by jaffe-fly** [comment]
Is it support obb now？

**Reply by fcakyon** [reply]
Yes @jaffe-fly its supported as of the latest version: https://github.com/obss/sahi/releases/tag/0.11.20

**Comment by fcakyon** [comment, marked as answer]
Ultralytics YOLOv5U/YOLOv8/YOLO11 OBB models are supported with latest sahi release: https://github.com/obss/sahi/releases/tag/0.11.20

**Reply by newlyXK** [reply]
May I ask if there are detailed examples regarding Sahi's support for OBB detection? I am currently trying to use my model, but it does not work for rotated boxes.

**Reply by newlyXK** [reply]
I got it!thanks
